### PR TITLE
master <- development

### PR DESCRIPTION
**Pull request by kudjieRaymond**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+footer{
+	text-align: center;
+  margin-top: 10px;
+}
+.copyright{
+	margin-left: 200px;
+	margin-right: 200px;
+	margin-top: 10px;
+	margin-bottom: 10px;
```
Comment (JamesOkunlade) [inline style.css]: Lines 121, 122, 123 and 124 can be reduced to one line. margin: 10px 200px;

Comment (kudjieRaymond) [reply style.css]: so can we go ahead and merge the pull request or we have to submit a new pull request

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
Nice work on html-forms guys! I can see no errors in your html code which is great. There is a very minor change you should make on you css however, check my comment on the line.

I also noticed the commits were not evenly distributed amongst you two. You should always switch roles to give yourselves commits on a particular project.

Effect the change on your CSS and request for another code review. Add me as the reviewer so I can follow up on it. 

Happy coding! :sunglasses:

File: style.css (hunk 1 above)
Comment (kudjieRaymond) [reply style.css]: @JamesOkunlade   so can we go ahead and merge the pull request or we have to submit a code review after making that change

Comment (i4seeu) [reply style.css]: OK got it @JamesOkunlade

**Review by JamesOkunlade — APPROVED** [review]
Great guys! 
Remove the unneeded comment in line 68 of your CSS, merge development to master and fill you completion form. Here are your thumps up :+1: :+1:

**Pull request merged**